src/util/color/predefinedcolorpalettes.cpp

```diff
@@ -144,6 +144,21 @@ constexpr mixxx::RgbColor kVirtualDJTrackColorBlue(0x0000FF);
 constexpr mixxx::RgbColor kVirtualDJTrackColorFuchsia(0xFF00FF);
 constexpr mixxx::RgbColor kVirtualDJTrackColorWhite(0xFFFFFF);
 
+// Default Mixxx Key Color Palette
```

Comment:
https://www-datylon-com.cdn.ampproject.org/v/s/www.datylon.com/blog/data-visualization-for-colorblind-readers?amp_gsa=1&amp_js_v=a9&hs_amp=true&usqp=mq331AQIUAKwASCAAgM%3D#amp_tf=Von%20%251%24s&aoh=17198172132733&csi=1&referrer=https%3A%2F%2Fwww.google.com&ampshare=https%3A%2F%2Fwww.datylon.com%2Fblog%2Fdata-visualization-for-colorblind-readers

Interesting are the hue curves of different color blind people. So I guess in a follow up PR, we may offer the full hue curve and reduced versions for color blind people that is only using half of the hue range.

Reply:
We could provide preset palettes optimized for the different types of colorblindness. I think gradients for those already exist online. This would be fairly straightforward to implement once the requusted custom keypalettes are in place.